app/ios/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>0A2A.1</string>
```

Comment:
I think "C617.1" might be better. I read "0A2A.1" to mean that BC Wallet is itself a 3rd-party SDK which we're not. This is probably used by AFJ to store the tails file and other caching. It notes "This reason may only be declared by third-party SDKs." Thoughts?

[Ref Doc](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api)

Reply:
I agree with using `C617.1`

Reply:
Agree and fixed. I misunderstood this one